website/src/pages/downloads.tsx

```diff
@@ -103,18 +104,75 @@ const VersionSelect = () => {
     );
 };
 
-const Asset = ({ url }) => {
+const Asset = ({ urls }) => {
+    let asset: string;
+    let gpgSig: string;
+    let coCert: string;
+    let coSig: string;
+    let name: string = "Binary";
+
+    for (let url of urls) {
+        if (url.toLowerCase().includes(".deb")) {
+            name = "Debian Package";
+        } else if (url.toLowerCase().includes(".rpm")) {
+            name = "RPM Package";
+        } else if (url.toLowerCase().includes(".pkg") && url.toLowerCase().includes("linux")) {
+            name = "Arch Package";
+        }
+
+        if (url.toLowerCase().includes(".gpgsig")) {
+            gpgSig = url;
+            continue;
+        }
+        if (url.toLowerCase().includes(".pem")) {
+            coCert = url;
+            continue;
+        }
+        if (url.toLowerCase().includes(".sig")) {
+            coSig = url;
+            continue;
+        }
+        asset = url;
+    }
+
+    if (asset === undefined) {
+        return;
+    }
+
     const { selectedItem } = useOptions();
     return (
-        <div className="pagination-nav__item">
-            <a className="pagination-nav__link" href={url}>
-                <div className="pagination-nav__label">
-                    {AssetArchitecture(url).toUpperCase()}
+        <div className="card download-card">
+            <div className="pagination-nav__item">
+                <div className="card__header">
+                    <h5>
+                        <a href={asset}>
+                            {AssetArchitecture(asset).toUpperCase()} - {name}
+                        </a>
+                    </h5>
                 </div>
-                <div className="pagination-nav__sublabel">
-                    Version: {selectedItem}
+                <div className="card__body">
+                    <p className="text--center">
+                        Version: {selectedItem}
+                    </p>
                 </div>
-            </a>
+                <div className="card__footer">
+                    <div class="button-group button-group--block">
+                        <a class="button button--primary" href={asset}><span>Download</span></a>
+                        {
+                            gpgSig &&
+                            <a class="button button--secondary" href={gpgSig}><span>GPG Signature</span></a>
+                        }
+                        {
+                            coSig &&
+                            <a class="button button--secondary" href={coSig}><span>Cosign Signature</span></a>
+                        }
+                        {
+                            coCert &&
+                            <a class="button button--secondary" href={coCert}><span>Cosign Certificate</span></a>
+                        }
+                    </div>
```

Comment:
This changes the buttons to have round corners everywhere, which looks much nicer if the cards are resized
```suggestion
                    <div className="button-group button-group--block">
                        <a
                            className="button button--primary"
                            href={asset}
                            style={{
                                borderRadius: "var(--ifm-button-border-radius)",
                            }}
                        >
                            <span>Download</span>
                        </a>
                        {gpgSig && (
                            <a
                                className="button button--secondary"
                                href={gpgSig}
                                style={{
                                    borderRadius:
                                        "var(--ifm-button-border-radius)",
                                }}
                            >
                                <span>GPG Signature</span>
                            </a>
                        )}
                        {coSig && (
                            <a
                                className="button button--secondary"
                                href={coSig}
                                style={{
                                    borderRadius:
                                        "var(--ifm-button-border-radius)",
                                }}
                            >
                                <span>Cosign Signature</span>
                            </a>
                        )}
                        {coCert && (
                            <a
                                className="button button--secondary"
                                href={coCert}
                                style={{
                                    borderRadius:
                                        "var(--ifm-button-border-radius)",
                                }}
                            >
                                <span>Cosign Certificate</span>
                            </a>
                        )}
                    </div>
```

Reply:
Cool, I added a property to the existing CSS rule for these to do so. Thanks @JanMa!!